packages/DatePicker/src/components/Calendar/Calendar.js

```diff
@@ -10,17 +10,20 @@ import ArrowLeft from "@paprika/icon/lib/ArrowLeft";
 import ArrowRight from "@paprika/icon/lib/ArrowRight";
 import useI18n from "@paprika/l10n/lib/useI18n";
 
-import CalendarStyled, { DayTriggerStyle, CalendarHeaderStyled } from "./Calendar.styles";
+import ShortcutPanel from "../ShortcutPanel";
+import CalendarStyled, { DayTriggerStyle, MonthHeaderButtonStyled, ArrowIconStyles } from "./Calendar.styles";
 
 const propTypes = {
   /** Selected date in moment object */
   date: momentPropTypes.momentObj,
 
-  /** Possible date might be selected in moment object */
-  possibleDate: momentPropTypes.momentObj,
+  isOpen: PropTypes.bool.isRequired,
 
   /** Callback to fire when user select date */
   onSelect: PropTypes.func.isRequired,
+
+  /** Possible date might be selected in moment object */
```

Comment:
Not sure what this means.

Reply:
We want to rerender `react-date` calendar when the user is typing in the input. This prop is the parsing result(in moment object) from user's input, and I use it to rerender calendar

Reply:
What makes it _possible_ though? I probably don't mind the name, but this comment is the opportunity to explain that name choice.

Reply:
maybe `predictedDate` ? or `prediction` ?